api4/plugin.go

```diff
@@ -37,11 +37,11 @@ func (api *API) InitPlugin() {
 	api.BaseRoutes.Plugin.Handle("/enable", api.APISessionRequired(enablePlugin)).Methods("POST")
 	api.BaseRoutes.Plugin.Handle("/disable", api.APISessionRequired(disablePlugin)).Methods("POST")
 
-	api.BaseRoutes.Plugins.Handle("/webapp", api.APIHandler(getWebappPlugins)).Methods("GET")
+	api.BaseRoutes.Plugins.Handle("/webapp", api.APIHandler(getWebappPlugins, model.ScopeNoScope)).Methods("GET")
 
 	api.BaseRoutes.Plugins.Handle("/marketplace", api.APISessionRequired(getMarketplacePlugins)).Methods("GET")
 
-	api.BaseRoutes.Plugins.Handle("/marketplace/first_admin_visit", api.APIHandler(setFirstAdminVisitMarketplaceStatus)).Methods("POST")
+	api.BaseRoutes.Plugins.Handle("/marketplace/first_admin_visit", api.APISessionRequired(setFirstAdminVisitMarketplaceStatus)).Methods("POST")
```

Comment:
why did this change from `APIHandler` to `APISessionRequired`?

Reply:
Piggybacked semantical fix. This function was checking a session was available, but the API handler was not doing it. There are no functional problems, but I guess I should change it.